app/views/pages/_slot_info.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class="next_slot">
+
+  <p>Prochaine séance</p>
+  <p><i class="fa fa-calendar" aria-hidden="true"></i>  <%= l(slot.date, format: :long) %></p>
+  <p><i class="fa fa-clock-o" aria-hidden="true"></i>  <%= slot.start_at.strftime("%H:%M") %> - <%= slot.end_at.strftime("%H:%M") %></p>
+  <p><i class="fa fa-user-o" aria-hidden="true"></i>  <%= slot.users.count %> / <%= slot.course.capacity_max %></p>
+
+  <!-- Si le user est inscrit à la prochaine séance-->
+  <% if slot.users.include?(current_user) %>
+    <p>Inscrit</p>
+  <!-- Si le cours est complet-->
+  <% elsif slot.is_full? %>
+    <p>Complet</p>
+  <!-- Sinon-->
+  <% else %>
```

Comment:
dans ma tête on avait dit que on mettait sur la card "Inscrit" ou "Complet" et ensuite un call to action "Toutes les séances". Du coup on ne met pas un "S'inscrire" nan?